README.md

```diff
@@ -43,6 +43,12 @@ libpcap should be installed on Mac OS X by default.
 
 If `PCAP_LIBDIR` environment variable is set when building the crate, it will be added to the linker search path - this allows linking against a specific `libpcap`.
```

Comment:
Just a note:
I'd never change this env var name to `LIBPCAP_LIBDIR`. I want to keep backward compatibility.